.github/workflows/publish_docker_image.yml

```diff
@@ -5,11 +5,13 @@ on:
     types: [published]
 
 env:
-  REGISTRY: ghcr.io
-  IMAGE_NAME: ${{ github.repository }}
+  GitHubREGISTRY: ghcr.io
+  GitHubIMAGE_NAME: ${{ github.repository }}
+  AURORA_PROD_REGISTRY: auroraprodacr.azurecr.io
+  AURORA_PROD_IMAGE_NAME: /robotics/flotilla
```

Comment:
```suggestion
  GITHUB_REGISTRY: ghcr.io
  GITHUB_IMAGE_NAME: ${{ github.repository }}
  AURORA_PROD_REGISTRY: auroraprodacr.azurecr.io
  AURORA_PROD_IMAGE_NAME: /robotics/flotilla
```

Reply:
Good comment, resolved

Reply:
Nothing fancy about the Aurora pushes, just an internal place to store the containers